Dockerfile

```
@@ -69,18 +76,26 @@ v${OSTICKET_VERSION}/osTicket-v${OSTICKET_VERSION}.zip; \
     rm osTicket.zip; \
     mkdir /usr/local/src; \
     mv upload /usr/local/src/osticket; \
+    # Patch usage of GLOB_BRACE
+    patch --directory="/usr/local/src/osticket" --forward --strip=1 \
+        --input="/tmp/osticket-glob-brace.patch"; \
```

```suggestion
    patch -d /usr/local/src/osticket -Np1 -i /tmp/osticket-glob-brace.patch; \
```

No need to make this unnecessarily long and complicated.